### ISSUE TITLE
Feature request : Name the tables

### ISSUE THREAD

**Issue by Jacques-Olivier-Farcy** [dozingcat/Vector-Pinball#60]
Will it be possible to have a name for the tables ?

**Comment by tum34-dot** [comment]
Here are some suggestions for the names :
N°1: Classic
N°2: The loft
N°3: U-turn
N°4: Abacus
N°5: Dodecagram
N°6: Gravity
N°7: Spaceship.

**Comment by dozingcat** [comment]
Good idea. I've been thinking of improving the UI for selecting tables, since as more get adding cycling through them one at a time can be annoying. (I've often gone past the table I was trying to choose, and then have to go through all the others again). Having names could be helpful for that.
